demos/mantle_convection.py

```diff
@@ -0,0 +1,197 @@
+# Mantle convection modelling
+#############################
```

Comment:
Need this or else it won't render properly as a title.
```suggestion
============================
```